comparative_benchmark/jax_xla/benchmark_all.sh

```diff
@@ -34,25 +34,25 @@ VENV_DIR="${VENV_DIR}" PYTHON="${PYTHON}" source "${TD}/setup_venv.sh"
 unset WITH_CUDA
 
 declare -a GPU_BENCHMARK_NAMES=(
-  "models/RESNET50_FP32_JAX_.+/inputs/.+/expected_outputs/.+/target_devices/a2-highgpu-1g"
-  "models/BERT_LARGE_FP32_JAX_.+/inputs/.+/expected_outputs/.+/target_devices/a2-highgpu-1g"
-  "models/T5_LARGE_FP32_JAX_.+/inputs/.+/expected_outputs/.+/target_devices/a2-highgpu-1g"
-  "models/T5_4CG_LARGE_FP32_JAX_.+/inputs/.+/expected_outputs/.+/target_devices/a2-highgpu-1g"
+  "models/RESNET50_FP32_JAX_.+/inputs/.+/expected_outputs/.+"
```

Comment:
nit: Should `/inputs/.+/expected_outputs/.+` be removed in the regex?

Reply:
Good catch. Done